iOS/Speaker.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Speaker.swift
+//  BotTalk
+//
+//  Created by Jeong YunWon on 2016. 7. 7..
+//  Copyright © 2016년 youknowone.org. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+public class Speaker: NSObject, AVSpeechSynthesizerDelegate {
+    static let defaultSpeaker = Speaker()
+    var volume: Float = 1.0
+    var mute = false
+    var voice = AVSpeechSynthesisVoice(language: "ko-KR")     // set default language : korean
```

Comment:
Say shouldn't set korean as default language. We should guess system default language, but "en-US" will be ok for the first step.